pkg/rbac/actions.go

```diff
@@ -75,6 +75,7 @@ const (
 	ActionCapsuleAbortRollout = "capsule:abortrollout"
 	// Stop rollout
 	ActionCapsuleStopRollout = "capsule:stoprollout"
+	ActionCapsuleApplySpec   = "capsule:applyspec"
```

Comment:
Guess we should put this under deploy, as it is no longer its own endpoint

Reply:
And add it in default permissions

Reply:
Ah that one needs to be removed actually. I dropped the endpoint in favour of baking the functionality into Deploy